.travis.yml

```diff
@@ -5,5 +5,6 @@ script:
   - pip install pytest
   - python setup.py install
   - pytest
+  - python examples/readme_example.py
```

Comment:
I like the example! It's clear and I hope easy to follow. I do think we need to call the thing by it's name; e.g. something like `optimise_nonlinear`.

Reply:
I've called it `simple_nonlinear` for now.